docker-compose.yml

```
@@ -0,0 +1,42 @@
+services:
+  endpoint:
+    image: node:20.15
```

```suggestion
    image: node:22.14
```

Currently the active LTS